webconsoleapp-local.yaml

```diff
@@ -32,6 +32,8 @@ spec:
           value: https://localhost:{PORT_3SCALE}
         - name: SESSION_INSTANCE_DOMAIN
           value: .dns.podman
+        - name: FAKE_AUTHENTICATION
```

Comment:
You're removing this in an earlier patch and adding it back here. Do not remove it in the first place?

Reply:
GitHub UI does not play well with force pushes. We'll squash the history when the PR is ready.